js/fieldToConceptIdMapping.js

```diff
@@ -23,8 +23,33 @@ export default
     "healthcareProvider":827220437,
     "verification":821247024,
     "verified":197316935,
+    "yes": 353358909,
+    "no": 104430631,
     "consentVersion": 454205108,
     "hipaaVersion": 412000022,
+    "heardAboutStudyForm" : 142654897,
+
+    "heardAboutStudyList":{
```

Comment:
This `List` is not a list (array). Will `checkBoxes` or `heardAboutStudyCheckBoxes` a better name?

Reply:
Solved